### PR TITLE
Longest Palindromic Substring C++

### PR DESCRIPTION
**Pull request by devanshiv123**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Efficient program for Longest Palindromic Substring added.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#632

### CODE REVIEW

File: C-Plus-Plus/cp/LongestPalindromicSubstring.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			high++;
+		}
+
+	}
+	return str.substr(start,maxLen);
+}
+
+int main(){
+	string str;
+	cin>>str;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/LongestPalindromicSubstring.cpp:54]: Display suitable message before taking input

File: C-Plus-Plus/cp/LongestPalindromicSubstring.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+	string LongestPalindrome=LongestPalinSubstr(str);
+	cout<<LongestPalindrome<<endl;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/LongestPalindromicSubstring.cpp]: Along with this output, add some messages

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
Do the changes suggested

**Comment by devanshiv123** [comment]
@siddharth25pandey @plazzy99 Please review the PR again. I have done the changes.

**Review by vatsalkesarwani12 — APPROVED** [review]
LGTM 👍

**Comment by HarshCasper** [comment]
@devanshiv123 Please resolve the Merge Conflict before I merge it

**Comment by devanshiv123** [comment]
> @devanshiv123 Please resolve the Merge Conflict before I merge it

@HarshCasper Resolved merge conflicts. Please merge the PR.

**Comment by atarax665** [comment]
@devanshiv123 Resolve the merge conflict ASAP.

**Comment by HarshCasper** [comment]
I'm closing the PR due to no activity. If you would like to reopen this PR to be reviewed please join the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and drop me a message.

**Pull request closed without merging**